src/components/TransactionsTab/TransactionsTab.tsx

```diff
@@ -229,7 +300,25 @@ const TransactionsTab = ({
                   {t("wallet.completedTransactions").toUpperCase()}
                 </LegendLine>
               </Legend>
+              <IconBinContainer
+                onClick={handleSetIsSelectorOpen}
+                onMouseEnter={() => setIsTooltip(true)}
+                onMouseLeave={() => setIsTooltip(false)}
+              >
+                <Icon iconSize={1} name="bin" />
+              </IconBinContainer>
             </LegendContainer>
+            <StyledTooltip $isTooltip={isTooltip}>
+              {t("wallet.clearList")}
+            </StyledTooltip>
+            <SelectWrapper $isOpen={isSelectorOpen} ref={selectWrapperRef}>
```

Comment:
Put the conditional around the component like so:
```
           {isSelectorOpen && (
              <SelectWrapper ref={selectWrapperRef}>
                <StyledDropdown
                  selectedOption={unit}
                  options={translatedOptions}
                  onChange={handleClearTypeChange}
                  isOpen={isSelectorOpen}
                />
              </SelectWrapper>
            )}
```

Reply:
This way the `isOpen` in Dropdown is unnecessary.

Reply:
Yeah it wasn't necessary. I removed it